source/reference/options.txt

```diff
@@ -468,6 +474,40 @@ Authentication Options
    - ``none``, the effect is the same as setting
      ``authMechanismProperties=CANONICALIZE_HOST_NAME:false``.
 
+.. option:: --oidcFlows
```

Comment:
Let’s mention that this is a comma-separated list, i.e. you could have `--oidcFlows=auth-code,device-auth` (which I think we’d prefer over e.g. only `--oidcFlows=device-auth`, because we generally consider device-auth less secure).

Reply:
Done.